src/bp/ui-admin/src/telemetry.ts

```diff
@@ -112,84 +113,47 @@ export function checkInfoReceived() {
   return !_.includes(info, '')
 }
 
-export function getServerFeedback() {
-  const pkgStr = window.localStorage.getItem('packageToSend')
-  let packages: Array<object> = []
-  if (pkgStr !== null) {
-    packages = JSON.parse(pkgStr)
-  }
-  return packages
-}
+export async function sendServerPackage() {
+  try {
+    const { data: packages } = (await api.getSecured().get(serverUrl)).data
 
-export function feedback(pkg) {
-  axios
-    .post(serverUrl, pkg, corsConfig)
-    .then(res => {
-      const packages = getServerFeedback()
-      if (packages.indexOf(pkg) !== -1) {
-        packages.splice(packages.indexOf(pkg), 1)
-        window.localStorage.setItem('packageToSend', JSON.stringify(packages))
-      }
-      console.log(res)
-    })
-    .catch(err => {
-      const packages = getServerFeedback()
-      if (packages.indexOf(pkg) === -1) {
-        packages.push(pkg)
-        window.localStorage.setItem('packageToSend', JSON.stringify(packages))
-      }
-      console.log(err)
-    })
-}
+    const feedback = { events: [], status: '' }
 
-export function sendServerPackage() {
-  if (window.localStorage.getItem('packageToSend') === null) {
-    window.localStorage.setItem('packageToSend', JSON.stringify([]))
-  }
+    if (packages && packages.url && packages.events) {
+      const url = packages.url
+      const events = packages.events
 
-  const packages = getServerFeedback()
-  packages.forEach((value, index) => {
-    feedback(value)
-  })
+      events.map(obj => (obj.source = 'client'))
 
-  axios
-    .get(serverUrl, corsConfig)
-    .then(res => {
-      if (_.has(res, 'data')) {
-        const payload = res.data.payload
-        const url = res.data.url
-        axios
-          .post(url, payload, corsConfig)
-          .then(res => {
-            feedback({ status: 'OK', data: payload })
-          })
-          .catch(err => {
-            feedback({ status: 'INACCESSIBLE', data: payload })
-            console.log(err)
-          })
+      feedback.events = events.map(obj => obj.uuid)
+
+      try {
+        await axios.post(url, events, corsConfig)
```

Comment:
isn't url always going to be the endpoint ?

Reply:
For this one... I really can't give an answer. Chris made the server sends the package with the url of the destination, so i can't say why. You should speak with him to understand why